presets/TestSamples/4.2/tune/tuning_example_1 - Copy (2).txt

```diff
@@ -0,0 +1,117 @@
+# some tuning example 4
```

Comment:
Looking at the file name, should these files be checked in? ;-)

Reply:
I am not sure either. But I need some "samples" to test the configurator part with... Maybe I can leave them in my repo only.

Reply:
@mikeller  these presets samples are just to test indexer.js and it's automation. Plus for the configurator part to test functionality before we add some real presets.